src/quacc/calculators/espresso/espresso.py

```diff
@@ -101,11 +109,56 @@ def read_results(self, directory: Path | str) -> dict[str, Any]:
         dict
             The results dictionnary
         """
+
         results = read(Path(directory) / self.outputname, binary=self.binary)
         if "energy" not in results:
             results["energy"] = None
         return results
 
+    def _outdir_handler(self, parameters: dict[str, Any]) -> dict[str, Any]:
+        """
+        Function that handles the various outdir of espresso binaries.
+        If the user-supplied path are absolute, they are resolved and checked
+        against the scratch_dir and results_dir. If they are relative, they are
+        resolved against the scratch_dir and results_dir. If they are not in
+        either of those directories, they are ignored. This case is discouraged,
+        are encouraged to use quacc's scratch_dir and results_dir.
```

Comment:
I don't see any mention of the scratch_dir or results_dir in this function. Not sure how the docstring is related there.

Reply:
I tried updating this.